instrumentation/executors/README.md

```diff
@@ -0,0 +1,6 @@
+# Settings for the executors instrumentation
+
+| System property | Type | Default | Description |
+|---|---|---|---|
+| `otel.instrumentation.executors.include` | Boolean | `false` | Whether to include property names. |
```

Comment:
That's not right. This property lists class names of `Executor` subclasses that we will monitor in addition to the hard-coded list that we maintain.

Reply:
So it'd be whether to monitor those additional `Executor` subclasses? Which one are those? Asking because the property itself is a boolean, not a list.

Reply:
No, I'm pretty sure it's a list: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/executors/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/javaconcurrent/AbstractExecutorInstrumentation.java#L103
The property name is stored in a constant, maybe that's what caused the script to detect the wrong type?

Reply:
Fixed, thanks!